### ISSUE TITLE
Modify git commit SHA1 state label format

### ISSUE THREAD

**Issue by chrissimpkins** [source-foundry/font-v#33]
Will be revised to the following format:

```
[abcd123]
[abcd123]-dev
[abcd123]-release
```

from 

```
abcd123
abcd123-dev
abcd123-release
```

as part of the new version specification that is in development.  This facilitates parsing of the SHA1 hash string and elimination of duplicate substrings (see #7)

See: https://github.com/source-foundry/font-v/issues/19#issuecomment-347990953

**Comment by chrissimpkins** [comment]
Slated to be part of v0.5.0 release, currently implemented in `dev` branch.

**Comment by chrissimpkins** [comment]
released in v0.5.0
